aptos-move/framework/aptos-framework/sources/account.spec.move

```diff
@@ -1,6 +1,145 @@
 spec aptos_framework::account {
+    
+    spec module {
+        pragma verify = true;
+        pragma aborts_if_is_strict;
+    }
+
     spec create_signer {
-        // TODO: temporary mockup.
         pragma opaque;
+        aborts_if [abstract] false;
+        ensures [abstract] signer::address_of(result) == addr;
+    }
+
+    spec initialize {
+        aborts_if !system_addresses::is_aptos_framework_address(signer::address_of(aptos_framework));
+        aborts_if exists<OriginatingAddress>(signer::address_of(aptos_framework));
+        ensures exists<OriginatingAddress>(signer::address_of(aptos_framework));
+    }
+
+    spec create_account {
+        pragma aborts_if_is_partial;
+        aborts_if exists<Account>(new_address);
+        aborts_if new_address == @vm_reserved || new_address == @aptos_framework || new_address == @aptos_token;
```

Comment:
I wonder whether this is actually a (small) bug and this should be any of the system addresses from 0x0 to 0xf. @movekevin

Reply:
Ping on this one @movekevin -- is create_account allowed on other system addresses?

Reply:
Technically yes it only prevents creating these 3 currently. However, since in genesis, we do pre-create all those accounts from 0x1-0xa, attempts to recreate them would fail. IMO, it's not critical to check against them here in the specs.

Reply:
> Technically yes it only prevents creating these 3 currently: https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/sources/account.move#L148. However, since in genesis, we do pre-create all those accounts from 0x1-0xa, attempts to recreate them would fail. IMO, it's not critical to check against them here in the specs.

The specs should just reflect what the implementation does, so the question is about the implementation, seems not to abort on other system addresses.